automation/release-bumper/release-bumper.sh

```diff
@@ -28,6 +28,10 @@ function main {
   update_versions
 
   echo INFO: Updating image digest list...
+  ( \
+    cd ./tools/digester && \
+    go build . \
+  )
```

Comment:
is it really necessary to split it into 4 lines? what about?
```suggestion
cd tools/digester && go build .
```

Reply:
It's more readable